package.json

```diff
@@ -2,7 +2,7 @@
   "name": "chartjs-plugin-datalabels",
   "homepage": "https://chartjs-plugin-datalabels.netlify.app",
   "description": "Chart.js plugin to display labels on data elements",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
It seems quite clear but are we sure that these changes are not breaking for Chart.js v3 users?

Reply:
Since there are no changes to the plugin code, yes its not breaking